spec/index.bs

```diff
@@ -624,12 +612,18 @@ GPUBuffer includes GPUObjectBase;
     : <dfn>\[[state]]</dfn> of type [=buffer state=].
     ::
         The current state of the {{GPUBuffer}}.
+
+    : <dfn>\[[readyAfterFenceID]]</dfn> of type [=unsigned long long=].
+    ::
+        The ID of the fence that must have completed before this buffer can
+        change {{[[state]]}} to "ready".
 </dl>
 
 Each {{GPUBuffer}} has a current <dfn dfn>buffer state</dfn> which is one of the following:
 
+ - "<dfn dfn for="buffer state">ready</dfn>" where the {{GPUBuffer}} is available for {{GPUBuffer/map()}} or {{GPUQueue.submit()}}.
  - "<dfn dfn for="buffer state">mapped</dfn>" where the {{GPUBuffer}} is available for CPU operations.
- - "<dfn dfn for="buffer state">unmapped</dfn>" where the {{GPUBuffer}} is available for GPU operations.
+ - "<dfn dfn for="buffer state">executing</dfn>" where the {{GPUBuffer}} is in-use for GPU operations.
```

Comment:
Is the transition from "executing" to "ready" defined by something user does (e.g. waiting on a fence), or happening behind the users back?